move_base/src/move_base.cpp

```diff
@@ -1181,4 +1181,46 @@ namespace move_base {
       controller_costmap_ros_->stop();
     }
   }
+
+  bool MoveBase::getRobotPose(tf::Stamped<tf::Pose>& global_pose, costmap_2d::Costmap2DROS* costmap)
```

Comment:
ABI in MoveBase doesn't matter, for Indigo+Kinetic -- let's add the name & transform_tolerance to this function signature, and then hard code them above (not great, but avoids ABI break). For Lunar -- let's go ahead and put this PR just this way (with the Costmap2DROS ABI change)